packages/execution-environments/src/common/sortParams.ts

```diff
@@ -2,6 +2,15 @@
 
 import { JSONRPCParams } from '../__GENERATED__/openrpc';
 
+/**
+ * Deterministically sort JSON-RPC parameter keys.
+ *
+ * @param method - The JSON-RPC method.
```

Comment:
This may need to be clarified. The method here is a method object that specifies the format of the expected JSON-RPC params. See: https://github.com/MetaMask/snaps-skunkworks/blob/main/packages/execution-environments/src/common/BaseSnapExecutor.ts#L111-L129